lale/operators.py

```diff
@@ -1357,6 +1362,13 @@ def __getattr__(self, name: str) -> Any:
                 return "classifier"  # satisfy sklearn.base.is_classifier(op)
             elif self.is_regressor():
                 return "regressor"  # satisfy sklearn.base.is_regressor(op)
+        if name == "classes_":
```

Comment:
The order of the checks here seems backwards if an impl explicitly provides a `classes_` property, we should probably use it rather then the one (implicitly) provided by the estimator.

Reply:
Good point. For my use cases, I first wanted to consider post-estimator mitigators (which currently rely on the estimator for `classes_`) followed by in-estimator mitigators (which have `classes_` defined on the impl). A better fix might be to change post-estimators so that `classes_` is defined directly on the impl and then reduce this to one case.